db/migrate/20230125212411_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.text :email, null: false
```

Comment:
the `text` type is for long strings
```suggestion
      t.string :email, null: false
```